utils.py

```diff
@@ -1,4 +1,9 @@
 def header():
     print("==============================================")
     print("Dijkstra Inn: The Best Inn in Town")
-    print("==============================================\n")
\ No newline at end of file
+    print("==============================================\n")
+
+
+def executeQuery(connector, query):
```

Comment:
util command for executing a query and returning the answer

Reply:
Would recommend wrapping in a try catch if you are working with a query that can fail